training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -56,11 +61,10 @@ object AdditiveModelTrainer {
                                    linfinityCap: Double,
                                    threshold: Double,
                                    epsilon: Double, // epsilon used in epsilon-insensitive loss for regression training
-                                   initModelPath: String,
-                                   linearFeatureFamilies: java.util.List[String],
-                                   priors: Array[String],
-                                   nDTreePipelineParams: NDTreePipelineParams,
-                                   classWeights: Map[Int, Float])
+                                   init: InitParams,
+                                   shuffle: Boolean,
+                                   classWeights: Map[Int, Float],
+                                   checkPointDir: String)
```

Comment:
we are adding more and more parameters here. you may want to write some comments to describe what are these